evap/student/templates/student_index_semester_evaluations_list.html

```diff
@@ -138,13 +138,13 @@
                                     <td>
                                         {% if not semester.results_are_archived %}
                                             {% if evaluation.is_single_result %}
-                                                {% if evaluation.state == 'published' %}
+                                                {% if evaluation.state == evaluation.State.PUBLISHED %}
                                                     <span data-toggle="tooltip" data-placement="top" title="{% trans 'Individual participants are not stored for single results. The number shown here is the number of votes received.' %}">
                                                         {% include 'progress_bar_fill.html' with icon='user' total=evaluation.single_result_rating_result.count_sum fill=100 %}
                                                     </span>
                                                 {% endif %}
                                             {% else %}
-                                                {% if evaluation.state == 'evaluated' or evaluation.state == 'reviewed' or evaluation.state == 'published' %}
+                                                {% if evaluation.state > evaluation.State.IN_EVALUTION %}
```

Comment:
It's dangerous to go alone. Here, take this: "A"

Did no test catch this? Feels risky...

Reply:
Actually, this is how I avoided passing more template data into `render`, because whenever there is an evaluation, I thought "why wouldn't I just access the class member through the instance?" (or maybe it was someone else's idea, I don't remember). Does it not work? I thought I tested this view ... :thinking:

Reply:
I'm fine with using `evaluation.State.WHATEVER`. I just wanted to point out that you wrote `EVALUTION` instead of `EVALUATION` -- this definitely should fail. Keep in mind: The template renderer will simply ignore all errors, it would just act as if the condition was false.